scripts/clean_osm_data.py

```diff
@@ -557,12 +582,7 @@ def _parse_float(x, ret_def=0.0):
             return ret_def
 
     # cables requirement for circuits calculation
-    cables_req = {
-        "50": 3,
-        "60": 3,
-        "16.7": 2,
-        "0": 2,
-    }
+    cables_req = {"50": 3, "60": 3, "16.7": 2, "0": 2}
 
     def _basic_cables(f_val, cables_req=cables_req, def_circ=2):
```

Comment:
Since we are here, could you replace the default value into 3 as most of the lines do have 3 cables?

```suggestion
    def _basic_cables(f_val, cables_req=cables_req, def_circ=3):
```